src/tests/ftest/cart/test_group.c

```diff
@@ -204,6 +204,25 @@ client_cb_common(const struct crt_cb_info *cb_info)
 		sem_post(&test_g.t_token_to_proceed);
 		D_ASSERT(rpc_req_output->bool_val == true);
 		break;
+	case TEST_OPC_SWIM_STATUS:
+		rpc_req_input = crt_req_get(rpc_req);
+		if (rpc_req_input == NULL)
+			return;
+		rpc_req_output = crt_reply_get(rpc_req);
+		if (rpc_req_output == NULL)
+			return;
+		if (cb_info->cci_rc != 0) {
+			D_ERROR("rpc (opc: %#x) failed, rc: %d.\n",
+				rpc_req->cr_opc, cb_info->cci_rc);
+			D_FREE(rpc_req_input->name);
```

Comment:
you shouldnt be freeing input fields of RPC

Reply:
581ebf161f1f6ab82a4ba2b619dbf33a31e5b92d

Reply:
422f0da